portfolio/src/main/webapp/notes.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Notes</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+
+    <h1>List of notes</h1>
+    <p>Notes/Questions/Comments I had while going through my webpage</p>
+
+    <p>
+        Webpage updates in real time only files that have been previously staged and comitted
+        <br>
+        However, that doesn't apply to unstaged files (a.k.a. have never been staged)
+    </p>
+
+    <p>
+        ADD HTML TAGS TO .html FILE
+    </p>
+
+    <p>
+        So I can use "< p > < / p >" for paragraphs, but theoretically I could just write like this w/o tags if I just want single lines.
```

Comment:
That's not a good idea. Web browsers contain a great deal of code to do their best with non-conforming text input but conforming input will

* parse faster
* render correctly on more browsers
* may rank higher in search results
* be possible to consistently interact with from JavaScript.